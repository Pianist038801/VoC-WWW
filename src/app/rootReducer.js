import {combineReducers} from 'redux';
import flashMessages from './reducers/flashMessages';
import auth from './reducers/auth';
import dashboardReducer from './reducers/dashboardReducer';
import global from './reducers/global';
export default combineReducers({flashMessages, auth, dashboardReducer, global});
