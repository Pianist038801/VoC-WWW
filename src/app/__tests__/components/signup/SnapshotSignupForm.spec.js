import React from 'react';
import { shallow, configure, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import { Provider } from 'react-redux';
import rootReducer from '../../../rootReducer';
import thunk from 'redux-thunk';
import { Router, browserHistory } from 'react-router';
import { createStore, applyMiddleware, compose } from 'redux';
import { SignupForm } from '../../../components/signup/SignupForm';
import SignupFormComponent from '../../../components/signup/SignupForm';
import { Link } from 'react-router';
import 'react-datepicker/dist/react-datepicker.css';
import sinon from 'sinon';
import renderer from 'react-test-renderer';
import Loader from 'react-loader';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });
const store = createStore(rootReducer, compose(applyMiddleware(thunk), window.devToolsExtension ? window.devToolsExtension() : f => f));

it('matches with the last snapshot', () => {
  let wrapper = shallow(<SignupForm userSignupRequest={() => {}} addFlashMessage={() => {}} isUserExists={() => {}} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
