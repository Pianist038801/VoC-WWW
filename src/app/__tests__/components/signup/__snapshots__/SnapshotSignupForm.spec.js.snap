// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches with the last snapshot 1`] = `
<form
  className="sign"
  onSubmit={[Function]}
>
  <h1
    className="loginText"
  >
    Sign up
  </h1>
  <TextFieldGroup
    checkUserExists={[Function]}
    field="username"
    label="Username"
    onChange={[Function]}
    type="text"
    value=""
  />
  <TextFieldGroup
    checkUserExists={[Function]}
    field="email"
    label="Email"
    onChange={[Function]}
    type="text"
    value=""
  />
  <TextFieldGroup
    field="password"
    label="Password"
    onChange={[Function]}
    type="password"
    value=""
  />
  <TextFieldGroup
    field="passwordConfirmation"
    label="Confirm password"
    onChange={[Function]}
    type="password"
    value=""
  />
  <div
    className="form-group"
  >
    <button
      className="btn btn-primary btn-lg"
      disabled={false}
    >
      Sign up
    </button>
  </div>
</form>
`;
